schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -0,0 +1,284 @@
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.HealthcareApis/services"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2019-09-16"
+          ]
+        },
+        "kind": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "fhir",
+                "fhir-Stu3",
+                "fhir-R4"
+              ]
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The kind of the service."
+        },
+        "location": {
+          "type": "string",
+          "description": "The resource location."
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The resource tags."
+        },
+        "etag": {
+          "type": "string",
+          "description": "An etag associated with the resource, used for optimistic concurrency when editing it."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/ServicesProperties"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The properties of a service instance."
+        }
+      },
+      "required": [
+        "name",
+        "type",
+        "apiVersion",
+        "kind",
+        "location",
+        "properties"
+      ],
+      "description": "Microsoft.HealthcareApis/services"
+    }
+  },
+  "definitions": {
+    "ServiceAccessPolicyEntry": {
+      "type": "object",
+      "properties": {
+        "objectId": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^(([0-9A-Fa-f]{8}[-]?(?:[0-9A-Fa-f]{4}[-]?){3}[0-9A-Fa-f]{12}){1})+$"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "An Azure AD object ID (User or Apps) that is allowed access to the FHIR service."
+        }
+      },
+      "required": [
+        "objectId"
+      ],
+      "description": "An access policy entry."
+    },
+    "ServiceAuthenticationConfigurationInfo": {
+      "type": "object",
+      "properties": {
+        "authority": {
+          "type": "string",
```

Comment:
The original format has the following. Just wondering is schema link is not required anymore?
"authority": {
          "oneOf": [
            {
              "type": "string",
              "description": "The authority url for the service"
            },
            {
              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
            }
          ],